rm-datasource/src/main/java/io/seata/rm/datasource/undo/parser/ProtostuffUndoLogParser.java

```diff
@@ -60,7 +60,7 @@ public String getName() {
 
     @Override
     public byte[] getDefaultContent() {
-        return new byte[0];
+        return new byte[1];
```

Comment:
why is `new byte[1]`, not `encode(new BranchUndoLog());`

Reply:
OK, I haven't noticed the decode method, closing this pr.